src/main/java/picard/vcf/FixVcfHeader.java

```diff
@@ -121,41 +114,54 @@ public static void main(final String[] args) {
         if (HEADER != null) IOUtil.assertFileIsReadable(HEADER);
         IOUtil.assertFileIsWritable(OUTPUT);
 
-        final VCFFileReader reader = new VCFFileReader(INPUT, false);
+        final VCFFileReader reader     = new VCFFileReader(INPUT, false);
+        final VCFHeader existingHeader = reader.getFileHeader();
 
         final VCFHeader outHeader;
         if (HEADER != null) { // read the header from the file
             final VCFFileReader headerReader = new VCFFileReader(HEADER, false);
+            final VCFHeader inputHeader      = headerReader.getFileHeader();
             if (ENFORCE_SAME_SAMPLES) {
-                outHeader = headerReader.getFileHeader();
-                enforceSameSamples(reader.getFileHeader(), outHeader);
+                outHeader = inputHeader;
+                enforceSameSamples(existingHeader, outHeader);
             }
             else {
-                outHeader = new VCFHeader(headerReader.getFileHeader().getMetaDataInInputOrder(), reader.getFileHeader().getSampleNamesInOrder());
+                outHeader = new VCFHeader(inputHeader.getMetaDataInInputOrder(), existingHeader.getSampleNamesInOrder());
             }
             CloserUtil.close(headerReader);
+            outHeader.getFilterLines()
+                    .stream()
+                    .filter(filterHeaderLine -> !existingHeader.hasFilterLine(filterHeaderLine.getID()))
+                    .forEach(filterHeaderLine -> log.info("FILTER line found in HEADER will be added to OUTPUT: " + filterHeaderLine));
```

Comment:
The other cases log `headerLine.getID()` but this case logs `headerLine`. Is that difference intentional?

Reply:
I blame IntelliJ for not finding this.  Thanks!